src/NuGet.Core/NuGet.Build.Tasks.Console/Program.cs

```diff
@@ -65,30 +56,39 @@ public static async Task<int> Main(string[] args)
                     securityInfo: null,
                     info: new AppDomainSetup
                     {
-                        ApplicationBase = arguments.MSBuildExeFilePath.DirectoryName,
-                        ConfigurationFile = Path.Combine(arguments.MSBuildExeFilePath.DirectoryName, "MSBuild.exe.config")
+                        ApplicationBase = msbuildFilePath.DirectoryName,
+                        ConfigurationFile = Path.Combine(msbuildFilePath.DirectoryName, "MSBuild.exe.config")
                     });
 
+                // Save the arguments as a string in the AppDomain to be parsed later since they can't be read from StandardInput again
+                //appDomain.SetData(nameof(StaticGraphRestoreArguments), argumentsJson);
```

Comment:
Should this be deleted before merging?   it doesn't look like debugging code useful to be left in.

Reply:
🤦‍♂️